documentation/limits.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Parser limits"
```

Comment:
Maybe call it limits - since its more than just Parser limits

Reply:
That's right it's about limits generally. Changed!